src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -182,16 +184,16 @@ trait RVD {
   protected def persistRVRDD(level: StorageLevel): PersistedRVRDD = {
     val localRowType = rowType
 
+    val persistCodec = CodecSpec.default
```

Comment:
This does a bit of compression which might not be quite what we want for cache().  Maybe make a new codec that just does Pack(Blocking(StreamBlock))?  (e.g. no lz4 or leb128)